torch/distributed/tensor/parallel/fsdp.py

```diff
@@ -43,6 +43,9 @@ def enable_2d_with_fsdp() -> bool:
     Return:
         A `bool` indicated whether extension registration succeeds or not.
     """
+
+    torch._C._log_api_usage_once("torch.distributed.tensor.parallel.enable_2d_with_fsdp")
```

Comment:
is this indention global? I think it should be local to `enable_2d_with_fsdp` function itself?

Reply:
The indention is same as L49.